### ISSUE TITLE
Identify places to track navigation to data viz through Tag Manager 

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#3766]
**What we're after:**
As part of the work to improve navigation to the data visualizations, we should [implement tag manager](https://github.com/fecgov/fec-cms/issues/3160). This will keep an eye on what users are looking, where they're clicking and give us more granular information on how users are moving through the site. This ticket is to identify specific places we want to track to track how are users move through the site. 

This ticket is blocked by the implementation and testing of tag manager on the site. 

### Related issues
- https://github.com/fecgov/fec-cms/issues/3763 Design navigation menus to improve path to landing page
- https://github.com/fecgov/fec-cms/issues/3764 Add links to visualizations in search results
- https://github.com/fecgov/fec-cms/issues/3765 Design a feature discovery tool
- https://github.com/fecgov/fec-cms/issues/3160 Tag Manager implementation
- https://github.com/fecgov/fec-cms/issues/3186 Tag Manager testing

### Completion criteria
- [ ] Identify places to track navigation to data viz for Tag Manager
- [ ] Open implementation issue, if necessary

**Comment by JonellaCulmer** [comment]
Closing in favor of #3948
